atlasdb-api/build.gradle

```diff
@@ -1,3 +1,4 @@
+apply from: "../gradle/publish-jars.gradle"
```

Comment:
nit: Maybe not in this PR but it would make more sense to apply the two internal gradle files and then the other plugins

```
apply from "../gradle/publish-jars.gradle"
apply from "../gradle/shared.gradle"

apply plugin: ...
```